site/_docs/datafiles.md

```diff
@@ -114,3 +114,33 @@ The organizations can then be accessed via `site.data.orgs`, followed by the fil
 </ul>
 {% endraw %}
 {% endhighlight %}
+
+## Example: Accessing a specific author
+
+Pages and posts can also access a specific data item. The example below shows how to access a specific item:
+
+`_data/people.yml`:
+{% highlight yaml %}
+dave:
+    name: David Smith
+    twitter: DavidSilvaSmith
+{% endhighlight %}
+
+The author can then be specified as a page variable in a post's frontmatter:
+
+{% highlight html %}
+{% raw %}
+---
+title: sample post
+author: dave
+---
+
+{% assign author = site.data.people.[page.author] %}
```

Comment:
Does it work without the added dot? It will likely break in Liquid 3's stricter lexer.